Resync `web-animations` from WPT Upstream

eerii opened the pull request:
#### ec30dff51652889e6b18e90dafde69d7421d2f42
<pre>
Resync `web-animations` from WPT Upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=304781">https://bugs.webkit.org/show_bug.cgi?id=304781</a>

Reviewed by Brandon Stewart.

Run `import-w3c-tests` for `web-animations`.
Needed to cover the CSS `math-*` properties from MathML Core.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/afce258ce1915a1a34522670d664031042e70f62">https://github.com/web-platform-tests/wpt/commit/afce258ce1915a1a34522670d664031042e70f62</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/accumulation-per-property-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/addition-per-property-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/interpolation-per-property-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/property-list.js:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-alternate.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-alternate.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-before-handlers.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-before-handlers.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-once.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-once.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-repeat.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-repeat.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-state.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-state.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/crashtests/get-timing-bad-pseudo-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/crashtests/set-timeline-undefined-progress.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/TimelineTrigger/constructor-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/AnimationTrigger/constructor-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/TimelineTrigger/constructor.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/AnimationTrigger/constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/TimelineTrigger/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/AnimationTrigger/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/responsive/background-color-animation-plus-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/responsive/background-color-animation-plus-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/responsive/background-color-animation-plus-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/responsive/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/305012@main">https://commits.webkit.org/305012@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/245610a3ffc65c3f3c5a700a3f5abf2ae9a84eb3

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/137160 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/9520 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/48447 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/144906 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/90133 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/10223 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/9647 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/104888 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/76504 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/140105 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/162/builds/7530 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/122912 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/85727 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/7167 "Passed tests") | [❌ 🧪 api-mac-debug](https://ews-build.webkit.org/#/builders/165/builds/4882 "Found 11 new API test failures: TestWebKitAPI.WKWebExtensionAPIExtension.GetViewsForMultipleTabs, TestWebKitAPI.MemoryFootprintThreshold.TestDelegateMethod, TestWebKitAPI.MessagePort.MessageToClosedPort, TestWebKitAPI.WKWebExtensionAPIStorage.SetAccessLevelTrustedContexts, TestWebKitAPI.WEBKIT.NavigationActionHasOpener12, TestWebKitAPI.NSResponderTests.ValidRequestorForSendAndReturnTypes, TestWebKitAPI.WKWebExtensionAPIAlarms.ClearAllAlarms, TestWebKitAPI.GetUserMedia.OriginAndWebArchivePermission, TestWebKitAPI.MediaLoading.LockdownModeHLS, TestWebKitAPI.PermissionsAPI.GeolocationPermissionDeniedPermanentlyAndGeolocationRequestedSinceLoad ... (failure)") | [✅ 🛠 wpe-cairo-libwebrtc](https://ews-build.webkit.org/#/builders/166/builds/5496 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/116519 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/41078 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/147662 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/9203 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/41641 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/113247 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/9221 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/7738 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/113578 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/28850 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/7086 "Passed tests") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/119171 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/63590 "The change is no longer eligible for processing.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/9251 "Built successfully") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/37225 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/8976 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/9192 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/9044 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

stwrt reviewed (approved):
LGTM if tests are passing.

eerii committed:
Resync `web-animations` from WPT Upstream
https://bugs.webkit.org/show_bug.cgi?id=304781

Reviewed by Brandon Stewart.

Run `import-w3c-tests` for `web-animations`.
Needed to cover the CSS `math-*` properties from MathML Core.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/afce258ce1915a1a34522670d664031042e70f62

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/accumulation-per-property-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/addition-per-property-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/interpolation-per-property-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/property-list.js:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-alternate.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-alternate.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-before-handlers.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-before-handlers.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-once.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-once.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-repeat.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-repeat.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-state.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/event-trigger-state.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-trigger/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/crashtests/get-timing-bad-pseudo-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/crashtests/set-timeline-undefined-progress.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/TimelineTrigger/constructor-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/AnimationTrigger/constructor-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/TimelineTrigger/constructor.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/AnimationTrigger/constructor.html.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/TimelineTrigger/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/web-animations/interfaces/AnimationTrigger/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/responsive/background-color-animation-plus-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/responsive/background-color-animation-plus-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/responsive/background-color-animation-plus-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/web-animations/responsive/w3c-import.log:

Canonical link: https://commits.webkit.org/305012@main
